src/schema/v2/homeView/sections/_ExampleSection.ts

```diff
@@ -239,6 +239,12 @@ export const ExampleSection: HomeViewSection = {
    */
   requiresAuthentication: false,
 
+  /*
+   * Whether the section should track item impressions.
+   *
+   */
+  trackItemImpressions: true,
```

Comment:
Since we recommend that devs actually copy this file to kick off a new section I wonder if we should actually default this to `false`? We discussed in Onyx office hrs that this results in very chatty analytics events, so we should deliberately opt in to this.